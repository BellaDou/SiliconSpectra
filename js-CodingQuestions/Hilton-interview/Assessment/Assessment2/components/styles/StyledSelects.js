import styled from "styled-components";

const StyledSelects = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: space-evenly;
`;

export default StyledSelects;
