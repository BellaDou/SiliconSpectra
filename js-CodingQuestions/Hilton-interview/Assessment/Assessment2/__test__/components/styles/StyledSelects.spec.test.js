import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import StyledSelects from "../../../components/styles/StyledSelects";

test("Should properly render style component", () => {
  const tree = renderer.create(<StyledSelects />).toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule("display", "flex");
  expect(tree).toHaveStyleRule("flex-direction", "row");
  expect(tree).toHaveStyleRule("justify-content", "space-evenly");
});
