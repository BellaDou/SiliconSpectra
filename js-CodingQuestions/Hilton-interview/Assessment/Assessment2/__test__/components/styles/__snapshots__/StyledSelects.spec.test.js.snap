// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should properly render style component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
}

<div
  className="c0"
/>
`;
